src/term/mod.rs

```diff
@@ -453,25 +453,24 @@ impl<'a> Iterator for RenderableCellsIter<'a> {
                 (cell, selected)
             };
 
-            let mut invert_selection = false;
             // Set custom selection color and selection text color, if they were provided.
+            let selection_background_color = self.config.selection_background_color();
```

Comment:
Moving this under the selection color comment and over the `let invert_selection` statement would probably make things look a little cleaner after all the other suggestions are applied.